czech/admin/forum_announcements.lang.php

```diff
@@ -5,7 +5,7 @@
  *
  */
 
-$l['forum_announcements'] = "Forum Announcements";
+$l['forum_announcements'] = "Oznámení";
```

Comment:
Taktéž opraveno. https://ctrlv.cz/1oJq 
Stejně, jak jsem se teď vrátil k překladu po pár dnech, tak každopádně se chci na to všechno podívat až to dopřeložím, kde to neodpovídá a nebo je nepřesné, tak o tom hodit řeč.
Případně to sjednotit s překladem, který už tu je.